tower-http/src/services/fs/serve_file.rs

```diff
@@ -74,7 +117,7 @@ impl ServeFile {
     }
 }
 
-impl<R> Service<R> for ServeFile {
+impl<ReqBody> Service<Request<ReqBody>> for ServeFile {
```

Comment:
Its a shame that this required a breaking change. I think for 0.2 we should audit all services and make sure they implement `Service<Request<B>>` so we can make changes like this in the future.